### ISSUE TITLE
Move checks to post report, before session info

### ISSUE THREAD

**Issue by timtrice** [workflowr/workflowr#110]
Before posting your issue, please search the existing issues (both "open" and "closed") to check that a similar feature has not already been requested.

1. Describe the suggested feature.

Ability to customize where the report appears in relation to the markdown document and session info. 

2. Describe why you think this feature is needed.

Report currently appears first before the report, below the title. This takes away from the reading order providing an unnecessary distraction. 

3. Tell us the version of workflowr you are using. Run `packageVersion("workflowr")` in R or RStudio, and copy & paste the output here.

`1.1.1`

**Comment by pcarbo** [comment]
@timtrice Thanks for your feedback. Where would you prefer to have the workflowr report?

**Comment by timtrice** [comment]
I think after the close of the document and before the session info would be a good default. But I'm wondering if it can be customized somehow. Perhaps in the _site.yml file?

**Comment by timtrice** [comment]
This is how I've customized it for a project I'm currently working on. But, I'm not personally a fan of it being hard-coded. 

```r
    lines_out <- c(header_lines,
                   lines_in[(header_end + 1):length(lines_in)],
                   "---",
                   "\nReport:",
                   report,
                   "---",
                   seed_chunk,
                   sessioninfo)
```

**Comment by jdblischak** [comment]
@timtrice I've made a bunch of changes to the workflowr report to make it less intrusive. It still contains lots of useful information, but it is collapsed by default. If you have time, could you please test out the new layout that is currently only available on the dev branch? I'd appreciate any feedback. You can install the version of the dev branch with:

```
devtools::install_github("jdblischak/workflowr@dev")
# After testing, re-install the current release
install.packages("workflowr")
```

Here are some screenshots of the report using different [Bootswatch themes](https://bootswatch.com/):

* A passing workflowr report using the cosmo theme (default)
![cosmo-pass](https://user-images.githubusercontent.com/1608317/50993648-35420c80-14e8-11e9-8037-c0f6234edfa2.gif)

* A failing workflowr report using the cosmo theme (default)
![cosmo-fail](https://user-images.githubusercontent.com/1608317/50993638-31ae8580-14e8-11e9-8c15-3bc697bf1621.gif)

* A passing workflowr report using the flatly theme
![flatly-pass](https://user-images.githubusercontent.com/1608317/50993635-2f4c2b80-14e8-11e9-97e6-9d14439dd44b.gif)

* A failing workflowr report using the flatly theme
![flatly-fail](https://user-images.githubusercontent.com/1608317/50993632-2ce9d180-14e8-11e9-9b37-37fe90e2fa0b.gif)

* A passing workflowr report using the united theme
![united-pass](https://user-images.githubusercontent.com/1608317/50993619-26f3f080-14e8-11e9-94c2-27092364ec1e.gif)

* A failing workflowr report using the united theme

![united-fail](https://user-images.githubusercontent.com/1608317/50993617-24919680-14e8-11e9-89e9-33404ed54997.gif)

**Comment by lazappi** [comment]
I like the new report format. Only comment I have is that the buttons for past versions of plots are a bit more intrusive now. Not sure if that is deliberate or not and it probably depends a bit on which theme you use.

**Comment by jdblischak** [comment]
> I like the new report format. Only comment I have is that the buttons for past versions of plots are a bit more intrusive now. Not sure if that is deliberate or not and it probably depends a bit on which theme you use.

@lazappi Thanks for the feedback! That's a good point. My intention wasn't necessarily to make them too prominent, but I did want them to be more consistently styled. I just pushed a commit that enables (and documents) how a user could style the workflowr buttons however they wish: https://github.com/jdblischak/workflowr/commit/1259e3af30c1e2f2be471d1fed22c786f45bdeb9

**Comment by jdblischak** [comment]
@timtrice This collapsed reproducibility report is now available in [workflowr 1.2.0](https://github.com/jdblischak/workflowr/releases/tag/v1.2.0). You can install it from the GitHub master branch or wait for it to be built and synced across the CRAN mirrors.
